src/helpers/ast/to-function-call.js

```diff
@@ -1,6 +1,7 @@
+import { types as t } from "babel";
```

Comment:
You shouldn't be importing from Babel. This should all be injected at runtime (at least in 5.x). This package doesn't even depend on Babel.